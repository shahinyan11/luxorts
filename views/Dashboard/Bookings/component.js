import { FormattedMessage } from 'react-intl';

import DashboardLayout from 'views/layouts/Dashboard';

const Bookings = () => <FormattedMessage id="shared.bookings" />;

Bookings.Layout = DashboardLayout;

export default Bookings;
