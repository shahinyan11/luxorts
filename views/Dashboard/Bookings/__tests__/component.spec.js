import { shallow } from 'enzyme';

import Bookings from '../component';

describe('Bookings component', () => {
  const component = shallow(<Bookings />);

  it('matches snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
