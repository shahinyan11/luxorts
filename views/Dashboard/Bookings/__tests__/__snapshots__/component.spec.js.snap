// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bookings component matches snapshot 1`] = `
<Memo(FormattedMessage)
  id="shared.bookings"
/>
`;
